tools/rstlint.py

```diff
@@ -221,8 +312,10 @@ def main(argv):
     else:
         for severity in sorted(count):
             number = count[severity]
-            print('%d problem%s with severity %d found.' %
-                  (number, number > 1 and 's' or '', severity))
```

Comment:
This is a standard way to write this.

Reply:
We follow PEP 8's "operators go in the front" revision from 2016. All code everywhere goes against this style because even PEP 8 used to advise the other way round. pycodestyle even has a warning about this (W503). I don't feel string interpolation is special enough to break the rules.